aspnet/tutorials/first-mvc-app/adding-model.rst

```diff
@@ -34,72 +31,64 @@ In the **Add Scaffold** dialog, tap **MVC Controller with views, using Entity Fr
 
 .. image:: adding-model/_static/add_scaffold2.png
 
-- Complete the **Add Controller** dialog
+- Complete the **Add Controller** dialog 
```

Comment:
Consider remove the bullet point for this text and just have the bulleted list below.